src/drivers/net/dwc_gmac/dwc_gmac.c

```diff
@@ -14,6 +14,7 @@
 #include <drivers/common/memory.h>
 #include <drivers/common/periph_memory_alloc.h>
 
+#include <asm-generic/dma-mapping.h>
```

Comment:
According to Linux documentation, dma interface is accessible with `<linux/dma-mappings.h>`, not `<asm-generic/dma-mappings.h>`

```
Part I - dma_API
----------------

To get the dma_API, you must #include <linux/dma-mapping.h>.  This
provides dma_addr_t and the interfaces described below.
```